storage/storage_test.go

```diff
@@ -0,0 +1,36 @@
+package storage
+
+import (
+	"context"
+	"os"
+	"testing"
+
+	"github.com/evergreen-ci/pail"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func makeTestStorage(t *testing.T, initDir string) Storage {
+	os.RemoveAll("../_bucketdata")
```

Comment:
should we `require.NoError` on these too?

Reply:
Yes.